src/test/java/org/gridsuite/modification/server/modifications/byfilter/assignment/BatteryModificationByAssignmentTest.java

```diff
@@ -0,0 +1,219 @@
+        assertEquals(2, getNetwork().getBattery(BATTERY_ID_1).getTargetQ(), 0);
+        ActivePowerControl activePowerControl1 = getNetwork().getBattery(BATTERY_ID_1).getExtension(ActivePowerControl.class);
+        assertNotNull(activePowerControl1);
+        assertEquals(2, activePowerControl1.getDroop(), 0);
+
+        assertEquals(80, getNetwork().getBattery(BATTERY_ID_2).getMaxP(), 0);
+        assertEquals(30, getNetwork().getBattery(BATTERY_ID_2).getTargetP(), 0);
+        assertEquals(80, getNetwork().getBattery(BATTERY_ID_3).getMaxP(), 0);
+        assertEquals(30, getNetwork().getBattery(BATTERY_ID_3).getTargetP(), 0);
+        assertEquals(80, getNetwork().getBattery(BATTERY_ID_4).getMaxP(), 0);
+
+        assertEquals(30, getNetwork().getBattery(BATTERY_ID_5).getMinP(), 0);
+        assertEquals(2, getNetwork().getBattery(BATTERY_ID_5).getTargetQ(), 0);
+        ActivePowerControl activePowerControl5 = getNetwork().getBattery(BATTERY_ID_5).getExtension(ActivePowerControl.class);
+        assertNotNull(activePowerControl5);
+        assertEquals(2, activePowerControl5.getDroop(), 0);
+
+        assertEquals(30, getNetwork().getBattery(BATTERY_ID_6).getMinP(), 0);
+    }
+
+    @Override
+    protected void assertAfterNetworkModificationDeletion() {
+        assertEquals(500, getNetwork().getBattery(BATTERY_ID_1).getMaxP(), 0);
+        assertEquals(80, getNetwork().getBattery(BATTERY_ID_1).getTargetQ(), 0);
+        ActivePowerControl activePowerControl1 = getNetwork().getBattery(BATTERY_ID_1).getExtension(ActivePowerControl.class);
+        assertNotNull(activePowerControl1);
+        assertEquals(1, activePowerControl1.getDroop(), 0);
+
+        assertEquals(2000, getNetwork().getBattery(BATTERY_ID_2).getMaxP(), 0);
+        assertEquals(200, getNetwork().getBattery(BATTERY_ID_2).getTargetP(), 0);
+        assertEquals(400, getNetwork().getBattery(BATTERY_ID_3).getMaxP(), 0);
+        assertEquals(300, getNetwork().getBattery(BATTERY_ID_3).getTargetP(), 0);
+        assertEquals(350, getNetwork().getBattery(BATTERY_ID_4).getMaxP(), 0);
+
+        assertEquals(50, getNetwork().getBattery(BATTERY_ID_5).getMinP(), 0);
+        assertEquals(140, getNetwork().getBattery(BATTERY_ID_5).getTargetQ(), 0);
+        ActivePowerControl activePowerControl5 = getNetwork().getBattery(BATTERY_ID_5).getExtension(ActivePowerControl.class);
+        assertNotNull(activePowerControl5);
+        assertEquals(4, activePowerControl5.getDroop(), 0);
+
+        assertEquals(200, getNetwork().getBattery(BATTERY_ID_6).getMinP(), 0);
+    }
+
+    @Override
+    protected IdentifiableType getIdentifiableType() {
+        return IdentifiableType.BATTERY;
+    }
+
+    @Override
+    protected EquipmentType getEquipmentType() {
+        return EquipmentType.BATTERY;
+    }
+
+    @Override
+    protected void assertResultImpacts(List<AbstractBaseImpact> impacts) {
+        // TODO later
```

Comment:
TODO when ?

What is the reason to delay this and keep it here ?

Reply:
DONE.. removed unnecessary override and implement only for LOAD which has a particular behaviour